src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/jupyterhub/files/jupyterhub/03-profiles.py

```diff
@@ -537,20 +550,43 @@ def preserve_envvars(spawner):
     return profile
 
 
+def parse_roles(data):
+    parsed_roles = {}
+
+    for role in data:
+        for group in role["groups"]:
+            group = str(group).replace("/", "")
```

Comment:
group names can consist on multiple `/`, e.g.

<img width="286" alt="Screenshot 2024-07-22 at 2 39 26 pm" src="https://github.com/user-attachments/assets/737979ec-d9c9-4ef2-84f5-bc747b47d9e5">

Reply:
Indeed, a subgroup name can also be exposed with dashes; thanks for the reminder. I will slice the first entry of the string instead.